changelogs/unreleased/3851-update-classifiers-in-setup-py.yml

```diff
@@ -0,0 +1,6 @@
+change-type: patch
+issue-nr: 3851
+description: Update python versions in the setup.py classifiers
+destination-branches: [master]
+sections:
+    upgrade-note: "{{description}}"
```

Comment:
Should this really go in the changelog?  (Is this useful information to the user of the package?)

Reply:
I guess not, I'll remove it